src/main/java/lindar/fonix/util/FonixDateUtil.java

```diff
@@ -0,0 +1,25 @@
+package lindar.fonix.util;
+
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Date;
+
+@Slf4j @NoArgsConstructor
+public class FonixDateUtil {
```

Comment:
Maybe?
```suggestion
public class FonixDateParser {
```
I have these terrible memories of big util classes doing too much, and since then, I'm avoiding the `util` in the name.

Reply:
I wasn't feeling inspired with naming yesterday, this is better 😂 Fixed in separate commit to rename all usages also 🙂